doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -31,6 +31,9 @@ The following sections provide detailed lists of changes by component.
 Application development
 =======================
 
+* Implemented new generic configuration scheme for build types in application development and updated several applications accordingly.
+  The now scheme uses application-specific :file:`prj.conf` files instead of build types selected through CMake build type variables.
```

Comment:
new not now.

Also, state the specific variable that is now longer used for this purpose.